mos.yml

```diff
@@ -435,9 +480,13 @@ conds:
       libs:
         - origin: https://github.com/mongoose-os-libs/adc
         - origin: https://github.com/mongoose-os-libs/wifi
+        - origin: https://github.com/mongoose-os-libs/pwm
+          variant: esp8266
```

Comment:
this is meaningless for a source-only library

Reply:
```suggestion
```